vars/kibanaTeamAssign.groovy

```diff
@@ -1,25 +1,23 @@
-def loadIngestionPipeline(ingestionPipelineName, title) {
+def generateCodeOwners(destination, title) {
   kibanaPipeline.bash("""
     source src/dev/ci_setup/setup_env.sh true
     yarn kbn bootstrap --prefer-offline
 
-    . src/dev/code_coverage/shell_scripts/assign_teams.sh '${ingestionPipelineName}'
+    . packages/kbn-dev-utils/src/code_ownership/shell_scripts/generate_code_owners.sh '${destination}'
```

Comment:
I don't think this script exists anymore

Reply:
Whoops!

Reply:
Thanks man!